frame/collective/src/lib.rs

```diff
@@ -650,6 +668,12 @@ pub mod pallet {
 	#[pallet::getter(fn prime)]
 	pub type Prime<T: Config<I>, I: 'static = ()> = StorageValue<_, T::AccountId, OptionQuery>;
 
+	/// True if we have upgraded so that AccountInfo contains three types of `RefCount`. False
+	/// (default) if not.
+	#[pallet::storage]
+	pub(super) type UpgradedToTripleRefCount<T: Config<I>, I: 'static = ()> =
```

Comment:
the name of the storage is not meaningful.
there is no triplerefcount upgrade in this pallet.
`UpgradedPalletPrefix` is more sensible to me

Reply:
Sounds good, Sure will modify to `UpgradedPalletPrefix`. 
I assumed `UpgradedToTripleRefCount` is coined after runtime version 
info like `Major.Minor.Subversion`.